nltk/tokenize/stanford_segmenter.py

```diff
@@ -70,9 +70,8 @@ def __init__(self,
                 searchpath=(), url=_stanford_url,
```

Comment:
if `slf4j` is not used, then the line 67 can be removed. Also, `self._SLF4J` is not needed.